src/tools/RelayGarbageCollection.js

```diff
@@ -1,69 +0,0 @@
-/**
- * Copyright 2013-2015, Facebook, Inc.
- * All rights reserved.
- *
- * This source code is licensed under the BSD-style license found in the
- * LICENSE file in the root directory of this source tree. An additional grant
- * of patent rights can be found in the PATENTS file in the same directory.
- *
- * @providesModule RelayGarbageCollection
- * @flow
- * @typechecks
- */
-
-'use strict';
-
-var RelayStoreData = require('RelayStoreData');
```

Comment:
Was this a public API? If so, let me know and I can add it back w/ deprecation warnings.